junebug/config.py

```diff
@@ -38,3 +38,7 @@ class JunebugConfig(Config):
     inbound_message_ttl = ConfigInt(
         "Maximum time allowed to reply to messages",
         default=60)
+
+    outbound_message_ttl = ConfigInt(
+        "Maximum time allowed for events to arrive for messages",
```

Comment:
Would it be worth mentioning the time unit in the description (similarly for the inbound ttl, sorry for not noticing earlier)?